File: Javascript/7kyu/growth-of-a-population.js
// Kata link: https://www.codewars.com/kata/growth-of-a-population

function nbYear(population, growthRate, aug, goal) {
    let years = 0;
    growthRate /= 100;

    while (population < goal) {
        population += parseInt(population * growthRate) + aug;
        years++;
    }

    return years;
}

File: C++/7kyu/growth-of-a-population.cpp
// Kata link: https://www.codewars.com/kata/growth-of-a-population


class Arge {
    public:
    static int nbYear(int population, double percent, int aug, int goal) {
        double parsedPercent = percent / 100;
        int years = 0;

        while (population < goal) {
            population += ((int) population * parsedPercent) + aug;
            years++;
        }

        return years;
    }
};